Throne/Models/LocationManager.swift

```diff
@@ -33,7 +33,7 @@ final class LocationManager: NSObject, ObservableObject, CLLocationManagerDelega
     func setupLocationManager() {
         // start tracking location
         locationManager.delegate = self
-        locationManager.distanceFilter = 20.0 // meters
+        locationManager.distanceFilter = 0.020 // kilometers
```

Comment:
This needs to be reverted, this a system API, not ours. This change would cause a location update for every more of your arm. Revert and don’t modify this file.

Reply:
Got it